tests/persistence/test_persistence_leverage.py

```diff
@@ -0,0 +1,638 @@
+                        5 hours = 5/24
+
+        interest: borrowed * interest_rate * time-periods
+                = 0.0019999999998453998 * 0.00050 * 1/24 = 4.166666666344583e-08  base
+                = 0.0019999999998453998 * 0.00025 * 5/24 = 1.0416666665861459e-07 base
+                = 0.0039999999996907995 * 0.00050 * 5/24 = 4.1666666663445834e-07 base
+                = 0.0039999999996907995 * 0.00025 * 1/24 = 4.166666666344583e-08  base
+    """
+
+    trade = Trade(
+        pair='ETH/BTC',
+        stake_amount=0.0009999999999226999,
+        amount=272.97543219,
+        open_rate=0.00001099,
+        open_date=datetime.utcnow() - timedelta(hours=0, minutes=10),
+        fee_open=fee.return_value,
+        fee_close=fee.return_value,
+        exchange='binance',
+        leverage=3.0,
+        interest_rate=0.0005,
+        interest_mode=InterestMode.HOURSPERDAY
+    )
+    # 10 minutes round up to 4 hours evenly on kraken so we can predict the them more accurately
+    assert round(float(trade.calculate_interest()), 22) == round(4.166666666344583e-08, 22)
+    trade.open_date = datetime.utcnow() - timedelta(hours=4, minutes=55)
+    # All trade > 5 hours will vary slightly due to execution time and interest calculated
+    assert float(round(trade.calculate_interest(interest_rate=0.00025), 14)
+                 ) == round(1.0416666665861459e-07, 14)
+
+    trade = Trade(
+        pair='ETH/BTC',
+        stake_amount=0.0009999999999226999,
+        amount=459.95905365,
+        open_rate=0.00001099,
+        open_date=datetime.utcnow() - timedelta(hours=4, minutes=55),
+        fee_open=fee.return_value,
+        fee_close=fee.return_value,
+        exchange='binance',
+        leverage=5.0,
+        interest_rate=0.0005,
+        interest_mode=InterestMode.HOURSPERDAY
+    )
+
+    assert float(round(trade.calculate_interest(), 14)) == round(4.1666666663445834e-07, 14)
+    trade.open_date = datetime.utcnow() - timedelta(hours=0, minutes=10)
+    assert float(round(trade.calculate_interest(interest_rate=0.00025), 22)
+                 ) == round(4.166666666344583e-08, 22)
+
+
+@pytest.mark.usefixtures("init_persistence")
+def test_update_open_order_lev(limit_lev_buy_order):
+    trade = Trade(
+        pair='ETH/BTC',
+        stake_amount=1.00,
+        open_rate=0.01,
+        amount=5,
+        fee_open=0.1,
+        fee_close=0.1,
+        interest_rate=0.0005,
+        exchange='binance',
+        interest_mode=InterestMode.HOURSPERDAY
+    )
+    assert trade.open_order_id is None
+    assert trade.close_profit is None
+    assert trade.close_date is None
+    limit_lev_buy_order['status'] = 'open'
+    trade.update(limit_lev_buy_order)
+    assert trade.open_order_id is None
+    assert trade.close_profit is None
+    assert trade.close_date is None
+
+
+@pytest.mark.usefixtures("init_persistence")
+def test_calc_open_trade_value_lev(market_lev_buy_order, fee):
```

Comment:
I'm not sure this test makes sense.

While it does add leverage over the "original" test - the method it tests has nothing to do with leverage - so i think we would not need this test.

Reply:
I was just checking that nothing changed